tests/commandline/test.py

```
@@ -30,12 +30,14 @@
 sourceDirectory = "."
 runtimeEnvironment = None
 
+
 class Test:
     def __init__(self, fileName, description, expected=True):
```

Ruff (0.14.1)

Unused method argument: `description`

(ARG002)